src/components/ui/Icons.jsx

```diff
@@ -0,0 +1 @@
+export { default as Heart } from '../../../public/images/feather/heart.svg';
```

Comment:
It would be more logical to name this file `icons.js`, as it isn't the definition of a React component.

Reply:
Ok I renamed the file to icon.js, but let it in same folder if it's ok ?